### ISSUE TITLE
texasholdem

### ISSUE THREAD

**Issue by LucasColas** [SirRender00/texasholdem#209]
Is there a plan to use this library for reinforcement learning ?

**Comment by SirRender00** [comment]
Certainly, this package can be used in RL perhaps by adding an adapter than can turn a TexasHoldEm object into a OpenAI Gym environment. That would require some work in writing the adapter.

I believe such an adapter has a place in this package.
